### ISSUE TITLE
No images shown in preview in forwarded email

### ISSUE THREAD

**Issue by Progressed36** [RainLoop/rainloop-webmail#1610]
Hi there,

Im having issues with images in forwarded e-mails. They dont show up, but is inserted into the email. when i save the email as .eml and open it with outlook they appear!

**RainLoop version, browser, OS:**
Im doing this with Rainloop 1.11.3 on a windows 10 with chrome, FF and IE

**Expected behavior and actual behavior:**
expected is that in preview mode it shows the image inserted into the email. 
actual behavior is that it does not.

**Steps to reproduce the problem:**
Forward email to a rainloop mailaccount.

**Logs or screenshots:**

**Comment by xals** [comment]
Hi,

I confirm the issue.

Here is my test:
- Having 2 accounts account1 and account2 on the same domain, both accessed using Rainloop
- Received an email in HTML format with inline image on account1
- Forwarded it to account2 using Thunderbird: the inline image is shown correctly in Rainloop
- Forwarded it to account2 using Rainloop: the inline imge it missing in Rainloop

I crawled a bit it the message source, the main difference I found is that on the message forwarded by Thunderbird, the image is in the same multipart as the html document but not in the one sent by Rainloop. It's been a bunch of years I haven't put my nose is mime/multipart so I am not sure it is related to this issue.

**Comment by QuijoteShin** [comment]
+1 
I believe the editor are changing the image and creating new ids?
Someone can point out which file handles the image parsing?

**Comment by andrewborell** [comment]
This is correct. There seems to be a problem forwarding emails from rainloop to rainloop. The problem is not the MIME types though. Its that the data-loaded attribute in the image tag needs to be set to false or stripped in the forwarded message.  Upon receipt the javascript does not execute to show the inline image because it thinks its already loaded. 

In this example, I used fiddler to check the outbound request body. Copied the &Text= element out, then decoded the html special character encoding.  The data-loaded attribute indicates its already loaded, which it is, but the browser is not going to display a custom data attribute as the image. That should be changed to a src attribute if the data is in fact already loaded. 

`<img width="802" height="174" style="width: 8.3583in;height: 1.8166in;opacity: 1" data-loaded="true" alt="" data-x-src-cid="image001.png@01D65AAA.E7176AA0" class="lazy" data-original="./?/Raw/&q[]=/_bWNRX3HX_KoNc_Wpu12mJ_30Hb3Wcvr4udRJ8qBnJgnEUd5T5YJJmUVDE8kA5Zz/View/&q[]=/5vQ2nIrO8O2aSZktrZ0bo59c8aMXUoDPNsvs" src="null" />`

I think basically one of two things needs to happen to resolve this problem. 

1. data-loaded="false"
2. data-original needs to be replaced with src=

You cannot say its loaded if its not loaded or the functions to display the image in the minified javascript (app.min.js) on the client side will not execute properly.  You can inspect the element and change data-original to src and the image will appear but your users will not like that solution.  

Waiting for guidance from rainloop devs to see what I can offer to help. I dont know where to begin with finding a place in code for forwarding messages to handle this.

**Comment by julyusito** [comment]
Helo,

Same issue here.

Somebody has a fix?

**Comment by julyusito** [comment]
Maybe some similar to this behavior?
They change a line into the nextcloud native mail app to show correctly the images.  Maybe similar to rainloop?

https://autoize.com/selecting-an-email-solution-to-integrate-with-nextcloud/

**Comment by yeupou** [comment]
Hello,

Bug seems to be still there in current version.

**Comment by lucentwolf** [comment]
...I'm also having this issue...
